instabot/api/api.py

```diff
@@ -46,7 +46,7 @@
 
 
 class API(object):
-    def __init__(self, device=None, base_path=""):
+    def __init__(self, device=None, base_path="", save_logfile=True, log_filename=None):
```

Comment:
is it possible to change path by doing
log_filename="path/instabot.log"
or is this only for filename change ?

Reply:
also work with the path :smile: